kubernetes_state/metadata.csv

```diff
@@ -41,6 +41,8 @@ kubernetes_state.node.pods_capacity,gauge,,,,The total pod resources of the node
 kubernetes_state.node.gpu.cards_allocatable,gauge,,,,The GPU resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.gpu.cards_allocatable
 kubernetes_state.node.gpu.cards_capacity,gauge,,,,The total GPU resources of the node,0,kubernetes,k8s_state.node.gpu.cards_capacity
 kubernetes_state.persistentvolumeclaim.status,gauge,,,,The phase the persistent volume claim is currently in,-1,kubernetes,k8s_state.persistentvolumeclaim.status
+kubernetes_state.persistentvolumeclaim.request_storage_bytes,gauge,,,,The amount of bytes requested by the persistent volume claim,-1,kubernetes,k8s_state.persistentvolumeclaim.request_storage_bytes
+kubernetes_state.persistentvolume.status,gauge,,,,The phase the persistent volume is currently in,-1,kubernetes,k8s_state.persistentvolume.status
```

Comment:
Let's inspire ourselves from the `nodes.by_condition` gauge, what about:

kubernetes_state.persistentvolumes.by_phase,gauge,,,,Number of persistent volume, to sum by phase and storageclass.,0,kubernetes,k8s_state.pv.by_phase